index.html

```diff
@@ -7,7 +7,17 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <header></header>
+    <header>
+        <nav>
+            <ul>
+                <li>Thing One</li>
+                <li>Thing Two</li>
+                <li>Thing Three</li>
+                <li>Thing Four</li>
+            </ul>
+        </nav>
+        <h1>STUFF N THINGS!!!!</h1>
```

Comment:
Update "N" to "&"

Reply:
As per our discussion, we are not making this change!

Reply:
Fine.